### ISSUE TITLE
Loss implementations

### ISSUE THREAD

**Issue by vict0rsch** [intel-isl/MiDaS#49]
Hi, 

I read in #43 that  you did not plan on releasing the training code. Can you still share the implementations you used for the losses?

Thanks!

**Comment by ranftlr** [comment]
You can find them linked in #2 and #33

**Comment by ranftlr** [comment]
Closing due to inactivity.
